### PR TITLE
[ticket/15528] Display style phpBB version

### PR DESCRIPTION
**Pull request by senky**
PHPBB3-15528

Checklist:

- [x] Correct branch: master for new features; 3.2.x, 3.1.x for fixes
- [x] Tests pass
- [x] Code follows coding guidelines: [master / 3.2.x](https://area51.phpbb.com/docs/master/coding-guidelines.html), [3.1.x](https://area51.phpbb.com/docs/31x/coding-guidelines.html)
- [x] Commit follows commit message [format](https://wiki.phpbb.com/Git#Commit_Messages)

Tracker ticket (set the ticket ID to **your ticket ID**):

https://tracker.phpbb.com/browse/PHPBB3-15528

### CODE REVIEW

File: phpBB/language/en/acp/styles.php [hunk 1]
```diff
@@ -80,6 +80,7 @@
 	'STYLE_UNINSTALL'			=> 'Uninstall',
 	'STYLE_UNINSTALL_DEPENDENT'	=> 'Style "%s" cannot be uninstalled because it has one or more child styles.',
 	'STYLE_UNINSTALLED'			=> 'Style "%s" uninstalled successfully.',
+	'STYLE_PHPBB_VERSION'		=> 'Style phpBB version',
```
Comment (Crizz0) [inline phpBB/language/en/acp/styles.php]: I would prefer an different wording and an explanation.
"Style for phpBB version" or similar.

Comment (DavidIQ) [reply phpBB/language/en/acp/styles.php]: Just want to point out that the label for the Extension details is _phpBB Version_ and is under a _Requirements_ section:
image

We should probably be consistent with styles, plus this affords styles authors the ability to add other requirements details that can be displayed in the future (even though they wouldn't be displayed right now, but they could be).

**Comment by Crizz0** [comment]
The test fails, because the 2. commit has a wrong commit message. Please juse `git rebase -i HEAD~2` to reword it to this format: https://area51.phpbb.com/docs/dev/3.2.x/development/git.html#commit-messages
